### ISSUE TITLE
Dashboard Page Pie Chart DIV is not showing the icon

### ISSUE THREAD

**Issue by sardarwaqasahmed** [akveo/blur-admin#399]
Dashboard page is not showing the icon for pie-charts. Image attached.
![screenshot_1](https://user-images.githubusercontent.com/9270627/40626002-e50cf5a0-62c6-11e8-815e-0f98e7a36d10.png)

**Comment by munavvard2** [comment]
thats hidden only for screen width between 1650px 400px ,
see here,
![screenshot from 2018-05-30 13-34-47](https://user-images.githubusercontent.com/10129780/40707452-583957b0-640e-11e8-8346-d53d37df4308.png)

**Comment by sardarwaqasahmed** [comment]
thanks munavvard2. By changing this css will it effect the rendering in other device?

**Comment by munavvard2** [comment]
yes , but it won't look good you have to manage width or fontsize
